cwms_radar_api/src/test/java/fixtures/TomcatServer.java

```diff
@@ -77,24 +76,23 @@ public TomcatServer(final String baseDir,
 
         Context blankToNull = tomcatInstance.addContext("", null);
 
-
         File radar = new File(radarWar);
         try{
             File existingRadar = new File(tomcatInstance.getHost().getAppBaseFile().getAbsolutePath(),contextName);
-            Files.walk(existingRadar.toPath()).sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
-            existingRadar.delete();
-            new File(existingRadar.getAbsolutePath()+".war").delete();
+            ExpandWar.delete(existingRadar);
+            ExpandWar.delete(new File(existingRadar.getAbsolutePath()+".war"));
+            ExpandWar.copy(radar, new File(tomcatInstance.getHost().getAppBaseFile(),"cwms-data.war"));
         } catch( Exception err) {
             System.out.println(err.getLocalizedMessage());
         }
 
-        Context context = tomcatInstance.addWebapp(contextName, radar.toURI().toURL());
+        //Context context = tomcatInstance.addWebapp(contextName, radar.toURI().toURL());
 
         if(authValve != null && realm != null)
```

Comment:
Can authValve just go away?  Seems like we don't use it anymore?

Reply:
yeah, but before I go that far I want to clean up other things.